<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoRemoveTagsDialog</class>
 <widget class="QDialog" name="GoRemoveTagsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Tags From Struct Field</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Remove Tags And Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="clearAllTagsRadioButton">
        <property name="text">
         <string>Clear All Tags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="removeAllOptionsRadioButton">
        <property name="text">
         <string>Clear All Tag Options</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="removeJsonTagRadioButton">
        <property name="text">
         <string>Remove JSON Tag</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="removeXmlTagRadioButton">
        <property name="text">
         <string>Remove XML Tag</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="removeCustomTagRadioButton">
        <property name="text">
         <string>Remove Custom Tag</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="customTaglineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="removeJsonOptionRadioButton">
        <property name="text">
         <string>Remove JSON Options</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="jsonOptionLineEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="removeXmlOptionRadioButton">
        <property name="text">
         <string>Remove XML Options</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="xmlOptionLineEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="removeCustomOptionRadioButton">
        <property name="text">
         <string>Remove Custom Tag Options</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="customOptionLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GoRemoveTagsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GoRemoveTagsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
